### PR TITLE
Add min_quntity to facilityinventorysummary and add migration

### PR DESCRIPTION
**Pull request by vasudevravula**
## Proposed Changes

- Add minimum quantity data to the inventory list.

### Associated Issue

[1721](https://github.com/coronasafe/care/issues/1721)
[6621](https://github.com/coronasafe/care_fe/issues/6612)

### Architecture changes

- Added min_quantity property to facilityinventorysummary model

## Merge Checklist

- [ ] Tests added/fixed
- [ ] Update docs in `/docs`
- [ ] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

**Comment by vasudevravula** [comment]
@rithviknishad @vigneshhari can you check this PR when you get a chance. Thank you.

**Review by rithviknishad — CHANGES_REQUESTED** [review]
No model changes required. Min qty field is already stored in the table, you just have to serialize that.

**Comment by vasudevravula** [comment]
@rithviknishad can you check now? Thank you

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@vasudevravula let me rephrase the issue's requirements for you more clearly

Currently we can set min. qty required for items from this page:

https://care.ohc.network/facility/915b2a99-3ee8-4ed4-b086-cfe3b4ca6148/inventory/min_quantity/set

image

And we can see the stocks available in the inventory at present from here

https://care.ohc.network/facility/915b2a99-3ee8-4ed4-b086-cfe3b4ca6148/inventory

image

The requirement is to show the set min. qty for those items in the manage inventory page itself. This would require to add an additional serializer READ ONLY field, that'll show this data along with listing the inventory items.

**Comment by vasudevravula** [comment]
@rithviknishad Thanks for the detailed explanation of the issue. I'm checking to make the changes, can you please assign the API and UI issues to me when you get a chance?
